collector/bfd.go

```diff
@@ -0,0 +1,135 @@
+package collector
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os/exec"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	bfdSubsystem = "bfd"
+
+	bfdCountLabels = []string{}
+	bfdPeerLabels  = []string{"local", "peer"}
+	bfdDesc        = map[string]*prometheus.Desc{
+		"bfdPeerCount":  colPromDesc(bfdSubsystem, "peer_count", "Number of peers detected.", bfdCountLabels),
+		"bfdPeerUptime": colPromDesc(bfdSubsystem, "peer_uptime", "Uptime of bfd peer in seconds", bfdPeerLabels),
+		"bfdPeerState":  colPromDesc(bfdSubsystem, "peer_state", "State of the bfd peer", bfdPeerLabels),
```

Comment:
Can you please add something like the below to the description?

(1 = Up, 0 = Down).

Reply:
Sure thing!

Reply:
Added the (1 = Up, 0 = Down) part 👍